README.md

```diff
@@ -54,36 +73,62 @@ if __name__ == "__main__":
     main(MyCharm)
 ```
 
-This charm does nothing, though, so you'll typically want to observe some Juju
-events, such as `start`:
+This charm does nothing, because the `MyCharm` class passed to the operator
+framework's `main` function is empty. Functionality can be added to the charm
+by instructing it to observe particular Juju events when the `MyCharm` object
+is initialized. In the example below we're observing the `start` event. The
+name of the event is a function of `self.on`, so in this case `self.on.start`
+is passed to `self.framework.observe`. This will look for method named
+`on_start` when that Juju event is triggered.
 
 ```python
 class MyCharm(CharmBase):
     def __init__(self, *args):
         super().__init__(*args)
-        self.framework.observe(self.on.start, self.on_start)
+        self.framework.observe(self.on.start, self)
 
      def on_start(self, event):
-        # Handle the event here.
+        # Handle the start event here.
 ```
 
 Every standard event in Juju may be observed that way, and you can also easily
-define your own events in your custom types.
+define your own events in your custom types. For example, to observe the
+`config-changed` Juju event:
```

Comment:
Can we please drop this extra example and respective introductory sentence?  This was added to document the the shorthand notation, but it's now simply repeating the same exact example described above.